torch/functional.py

```diff
@@ -744,199 +733,6 @@ def chain_matmul(*matrices):
     return torch._C._VariableFunctions.chain_matmul(matrices)
 
 
-def pstrf(a, upper=True, out=None):
```

Comment:
If we wanted to be even safer we could just make this a hard error and fully remove it in the next release. How long have these functions been deprecated for already?